eth2/types/src/beacon_state_tests.rs

```diff
@@ -0,0 +1,74 @@
+use crate::{
```

Comment:
Should we put this under a  #[cfg(test)] ? Or are we using the BeaconStateTestBuilder elsewhere?